openshift/app.dc.yaml

```diff
@@ -174,6 +192,44 @@ objects:
               name: ches-keycloak-config
           - configMapRef:
               name: ches-server-config
+          volumeMounts:
+            # mount log file location
+            - name: log-storage
+              mountPath: /var/log
+
+        # Fluent-bit sidecar
+        - image: fluent/fluent-bit:1.5
+          imagePullPolicy: Always
+          name: fluent-bit
+          resources:
+            requests:
+              cpu: "${SIDECAR_CPU_REQUEST}"
+              memory: "${SIDECAR_MEMORY_REQUEST}"
+            limits:
+              cpu: "${SIDECAR_CPU_LIMIT}"
+              memory: "${SIDECAR_MEMORY_LIMIT}"
+            readinessProbe:
+              httpGet:
+                path: /
+                port: 2020
+            livenessProbe:
+              httpGet:
+                path: /
+                port: 2020
```

Comment:
This is indented one level too far (should not be under `spec.containers.resources`. They're also missing the other probe parameters:
* failureThreshold
* initialDelaySeconds
* periodSeconds
* successThreshold
* timeoutSeconds